sql/updates/world/2016_03_19_00_world.sql

```diff
@@ -0,0 +1,3 @@
+-- Blinding Light
+DELETE FROM `spell_script_names` WHERE `spell_id` = 115750;
```

Comment:
If the TC contributors / developers agree, you could use the spell ID, at least if it is misplaced.